Diary/Controller/DiaryViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  Diary - DiaryViewController.swift
+//  Created by kaki, 레옹아범. 
+//  Copyright © yagom. All rights reserved.
+// 
+
+import UIKit
+
+final class DiaryViewController: UIViewController {
+    private var tableView: UITableView = UITableView()
+    private var diaryItems: [DiaryModel] = []
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = .systemBackground
+        tableView.dataSource = self
+        tableView.delegate = self
+        
+        configureTableView()
+        configureNavigationController()
+        parseSampleData()
+    }
+    
+    private func parseSampleData() {
+        guard let dataAsset = NSDataAsset(name: "sample") else { return }
+        
+        let decoder = JSONDecoder()
+        
+        do {
+            diaryItems = try decoder.decode([DiaryModel].self, from: dataAsset.data)
+        } catch {
+            print(error.localizedDescription)
+        }
+    }
+    
+    private func pushDetailViewController(_ item: DiaryModel?) {
```

Comment:
```suggestion
    private func pushDiaryDetailViewController(with diary: Diary?) {
```
로 쓰면 읽기 더 명확해질 거 같아요